tests/integration_tests.rs

```diff
@@ -756,6 +892,95 @@ fn config_read_arguments_from_file() {
         .stdout(predicate::eq("dummy-pager-from-config\n").normalize());
 }
 
+// ignore this test for now as cargo cache --clear only targets the default projects dir
```

Comment:
```suggestion
// ignore this test for now as bat cache --clear only targets the default projects dir
```

Reply:
I'm also not entirely sure I understand the reasoning here. Can't we use `BAT_CACHE_PATH` here?

Reply:
> To enable this test, I'd prefer the bat cache --clear function to clear the --target dir if specified, rather than the default dir.

I agree. That should be fixed (in a separate PR). It would be great if you could open a short ticket for this that also references this test here (because the `ignore` could then be removed).